app/uk/gov/hmrc/eeittadminfrontend/services/AuthService.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eeittadminfrontend.services
+
+import cats.data.Validated
+import play.api.Configuration
+import uk.gov.hmrc.eeittadminfrontend.models.{Email, LoginError}
+
+class AuthService(config: Configuration) {
+
+  def checkUser(email: Email): Validated[LoginError, Unit] = {
```

Comment:
I used either as neither of the results is invalid. they are both valid responses. didnt feel right labelling one as invalid when it is ok.